docs/resources/fgs_function.md

```diff
@@ -352,6 +363,45 @@ The `aliases` block supports:
 
 * `description` - (Optional, String) Specifies the description of the version alias.
 
+<a name="functiongraph_tactics_config"></a>
+The `tactics_config` block supports:
+
+* `cron_configs` - (Optional, List) Specifies the lists of scheduled policy configurations.
+  The [object](#functiongraph_cron_configs) structure is documented below.
+
+* `metric_configs` - (Optional, List) Specifies the lists of metric policy configurations.
+  The [object](#functiongraph_metric_configs) structure is documented below.
+
+<a name="functiongraph_cron_configs"></a>
+The `cron_configs` block supports:
+
+* `name` - (Required, String) Specifies the name of scheduled policy configuration.
+
+* `cron` - (Required, String) Specifies the cron expression.
+
+* `count` - (Required, Int) Specifies the number of reserved instance to which the policy belongs.
+
+-> The number of reserved instances must be greater than or equal to the number of reserved instances in the basic configuration.
+
+* `start_time` - (Required, Int) Specifies the effective timestamp of policy.
```

Comment:
Please specify the unit of the timestamp and give an example.

Reply:
fixed